runtime/doc/backers.txt

```diff
@@ -1,8 +1,15 @@
-# Bountysource Backers
+*backers.txt*          Nvim
 
-Thank you to everyone who backed our [Bountysource fundraiser](https://www.bountysource.com/teams/neovim/fundraiser)!
 
-### Your name and URL in BACKERS.md.
+                     NVIM REFERENCE MANUAL
+
+
+==============================================================================
+Fundraiser Backers
+
+Thank you to everyone who backed the original Nvim Fundraiser.
```

Comment:
I think this should be "Neovim" still? (Project, not editor?)

Reply:
Lol the test failed, and I was feeling lazy. I'll take a closer look to see if there's a way to ignore a file.

Reply:
Adjusted spell script a bit to allow ignoring files.

Reply:
Ah, I see. Not worth messing with the scripts over this, then.